proxy-node-translator/src/main/java/uk/gov/ida/notification/translator/logging/HubResponseTranslatorLogger.java

```diff
@@ -12,31 +8,6 @@
 
 public class HubResponseTranslatorLogger {
```

Comment:
We can rename this to `HubResponseAttributesHashLogger` since it's only used to log the attributes hash now

Reply:
Yes, will rename